packages/styles/scss/components/masthead/_masthead-l1.scss

```
@@ -38,11 +40,11 @@ $search-transition-timing: 95ms;
     }
   }
 
-  .#{$prefix}--masthead__l1-name {
+  .#{$prefix}--masthead__l1-name, :host(#{$dds-prefix}-masthead-l1-name){
     display: flex;
     height: 100%;
     background-color: $ui-02;
-    color: $text-04;
+    // color: $text-04;
```

I guess you meant to restore this CSS rule?

```suggestion
    color: $text-04;
```